### PR TITLE
[release-4.4] Bug 1810835: CreateContainerConfigError when obc is attached to deployment

### PR DESCRIPTION
**Pull request by gnehapk**
backport PR for https://github.com/openshift/console/pull/4574 and https://github.com/openshift/console/pull/4682

### CODE REVIEW

File: frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/modals/attach-deployment-obc-modal.tsx [hunk 1]
```diff
@@ -78,21 +70,21 @@ const AttachDeploymentToOBCModal = withHandlePromise((props: AttachDeploymentToO
...
           op: 'add',
-          path: `/spec/template/spec/containers/${i}/env/-`,
+          path: `/spec/template/spec/containers/${i}/envFrom/-`,
           value: secretMapRef,
         });
       } else {
         patch.push({
           op: 'add',
-          path: `/spec/template/spec/containers/${i}/env/-`,
+          path: `/spec/template/spec/containers/${i}/envFrom/-`,
```
Comment (afreen23) [inline frontend/packages/noobaa-storage-plugin/src/components/object-bucket-claim-page/modals/attach-deployment-obc-modal.tsx]: You missed here L92
```
  path: `/spec/template/spec/containers/${i}/env/-`,
```

**Comment by gnehapk** [comment]
@afreen23 @bipuladh Updated the PR. Please review

**Comment by afreen23** [comment]
@gnehapk  instead of creating a new commit it would be better if you create a cherrypick commit that will help patch managers and reviewers.

You have two commits :
9830057b59648f54e308e8e8f9a4ae3e7064a876
and
dbeb3c18f5d182a2151856769f365d7d27066240

I suggest you two send them both as a cherrypick commit. You can do 
`cherrypick -x 9830057b59648f54e308e8e8f9a4ae3e7064a876 dbeb3c18f5d182a2151856769f365d7d27066240`.

Thanks!

**Comment by gnehapk** [comment]
closing this PR as send another cherry pick PR https://github.com/openshift/console/pull/4687
/close

**Pull request closed without merging**